docs/code_push/release.md

```diff
@@ -45,3 +45,43 @@ Your next step is to upload the app bundle to the Play Store.
 See the following link for more information:
 https://support.google.com/googleplay/android-developer/answer/9859152?hl=en
 ```
+
+# Manage Releases
```

Comment:
Thoughts on changing adjusting the h1 at the top to be `# Releases` with an h2 for `## Publish a Release` and then we can make `Manage Releases` an h2 as well?

Reply:
Yeah, I like that. Done.